pkg/testutils/modulecr.go

```diff
@@ -23,6 +23,7 @@ const (
 	ModuleManagedCRName                = "template-operator-managed-resource"
 	ModuleDeploymentNameInNewerVersion = "template-operator-v2-controller-manager"
 	ModuleDeploymentNameInOlderVersion = "template-operator-v1-controller-manager"
+	NewModuleVersion                   = "2.4.2-e2e-test"
```

Comment:
this version better read from moduletemplate, it's more flexible in case we update the test moduletemplate later.